com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardProgram.hlsl

```diff
@@ -300,9 +309,12 @@ Varyings VertexStage(Attributes input)
     coefficients[4] = unity_SHBg;
     coefficients[5] = unity_SHBb;
     coefficients[6] = unity_SHC;
-    output.ambient = max(0.0, SampleSH9(coefficients, worldNormal));
+
+    v2f.ambient = max(0.0, SampleSH9(coefficients, envNormal));
 #else
-    output.ambient = ShadeSH9(float4(worldNormal, 1.0));
+#if defined(_SPHERICAL_HARMONICS)
```

Comment:
This check shouldn't be needed if you are just checking if _SPHERICAL_HARMONICS above.

```suggestion
```

Reply:
Good catch fixed.